src/Discovery/DiscoveryActionBar/components/OpenFillRequestFormButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { FileTextOutlined } from '@ant-design/icons';
+import { Popover, Button } from 'antd';
+
+const OpenFillRequestFormButton = ({
+    props,
+  }) => (
+  props.config.features.exportToWorkspace?.openFillRequestForm && (
```

Comment:
```suggestion
  props.config.features.exportToWorkspace?.enableFillRequestForm && (
```

Reply:
Change done, I will commit together once all of this is resolved